### ISSUE TITLE
first day of week

### ISSUE THREAD

**Issue by stereonom** [soyuka/dpicker#25]
Hi.

I set momentjs to my local translations. This works fine.
But the first day of the week isn't correct in dpicker. It's always sunday.
Any way to change this?

**Comment by soyuka** [comment]
Hmm, we rely on momentjs days sorting. You may try to change the days array (`dpicker.days = ['mon', 'tue'...]`) but there will be an issue because moment reads `0` as sunday. 

I'll investigate. You would like `monday` as first item?

**Comment by stereonom** [comment]
Yeah monday should be the first.

In moment-de.js there is a definition for the first day:
```
week : {
            dow : 1, // Monday is the first day of the week.
            doy : 4  // The week that contains Jan 4th is the first week of the year.
}
```
Perhaps you could check that value?

**Comment by soyuka** [comment]
May you check if the master branch works for you? Thanks!

**Comment by soyuka** [comment]
Landed in 3.1.4
